test/NuGet.Core.Tests/NuGet.Configuration.Test/PackageSourceMappingProviderTests.cs

```diff
@@ -410,5 +410,373 @@ public void AddOrUpdatePackageSourceMapping_WithAClearItem_WithANewPattern_AddIn
                 .Should().BeEquivalentTo(
                 File.ReadAllText(configPath1).Replace("\r\n", "\n"));
         }
+
+        [Fact]
+        public void SavePackageSourceMappings_WithNewMappings_Add()
```

Comment:
SavePackageSourceMappings_WithNewMappings_Add and SavePackageSourceMappings_WithUpdatedMappings_Update seem to have the exact same implementation.

Maybe you meant to just add new mappings in the first one and not remove existing ones?

Reply:
They are slightly different. SavePackageSourceMappings_WithNewMappings_Add() tests adding mappings to a whole new source. SavePackageSourceMappings_WithUpdatedMappings_Update() tests removing a mapping from a source and adding mappings to that same source

Reply:
Must have missed it. My bad.